### PR TITLE
Added demo images and coder badges to README.md

### PR DESCRIPTION
**Pull request by Pranav016**
# What is the change?
Added demo images and coder badges to README.md

# What does it fix/add?
Solves issue #4 and improves the README.md

# Submissions guide:
- [x] Have you followed the [Contribution guide](https://github.com/Neilblaze/Anyxz/blob/master/CONTRIBUTING.md)?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Neilblaze/Anyxz/pulls) for the same update/change?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you lint your code locally prior to submission?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <h3 align="center">Anyxz</h3>
 
   <p align="center">
-    Realtime Video Background Semantic Segmentation in Browser ⚡
+    <b>Realtime Video Background Semantic Segmentation in Browser ⚡ </b>
```
Comment (Neilblaze) [inline README.md]: Don't make it bold. The title should always be in bold. You can tweak with the font-size of the title.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,18 @@
 * [ml5 -> Unet](https://ml5js.org/reference/api-unet/)
 * [p5js](https://p5js.org/)
 
+<div align="center">
+
+[![forthebadge](https://forthebadge.com/images/badges/open-source.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/uses-html.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/uses-css.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com)
```
Comment (Neilblaze) [inline README.md]: Looks Good :v: 

Just a little bit of modification required. 

- Replace, `CSS` with `Bootstrap`
- Remove `Winter is Coming`.

File: README.md [hunk 3]
```diff
@@ -25,6 +25,18 @@
...
 
+<div align="center">
+
+[![forthebadge](https://forthebadge.com/images/badges/open-source.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/uses-html.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/uses-css.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com) 
+[![forthebadge](https://forthebadge.com/images/badges/makes-people-smile.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/powered-by-coffee.svg)](https://forthebadge.com) [![forthebadge](https://forthebadge.com/images/badges/winter-is-coming.svg)](https://forthebadge.com)
+
+</div>
+
+## Project Demo-
+![Demo1](assets/demo1.png)
```
Comment (Neilblaze) [inline README.md]: For now, the demo screenshot is not required. Once this project gets a good amount of UI revamp, then feel free to raise a PR & do that.

So, remove the screenshots.

File: README.md [hunk 4]
```diff
@@ -25,6 +25,18 @@
 * [ml5 -> Unet](https://ml5js.org/reference/api-unet/)
 * [p5js](https://p5js.org/)
 
+<div align="center">
```
Comment (Neilblaze) [inline README.md]: Shift this above, just below `View Demo · Report Bug · Request Feature`

File: README.md [hunk 5]
```diff
@@ -48,17 +60,19 @@ Now you're ready to do something awesome!
 
 
 <!-- ROADMAP -->
-## Roadmap
```
Comment (Neilblaze) [inline README.md:51]: add a single `<br />` above this & change the scaling to 0.5x i.e. `##` from `###`

**Review by Neilblaze — CHANGES_REQUESTED** [review]
Make these small changes & you're good to go! :rocket:

**Comment by Pranav016** [comment]
@Neilblaze Sure, I'll make all the suggested changes 👍

**Comment by Pranav016** [comment]
Added the Html5 and Bootstrap icons that look really good and cleaned up the code and the overall layout 👍

**Review by Neilblaze — APPROVED** [review]
@Pranav016 Looks great!
Thank you :smiley:

**Comment by Neilblaze** [comment]
@all-contributors please add @Pranav016 for doc

**Pull request merged**